docs/samples/drift-detection/alibi-detect/cifar10/cifar10_drift.ipynb

```diff
@@ -13,8 +13,9 @@
     "\n",
```

Comment:
<p>Is there a still default namespace scoped broker ingress or this is global now ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/kfserving/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Not any more. You need to create a Knative broker. So one further change could be to be the url of the logger required as there is no default that makes sense.

Reply:
Actually the above is not true. In more recent KNative eventing you need to add the "Sugar Controller" will will then react as before to annotations on the namespace to add a default Broker : https://knative.dev/docs/install/any-kubernetes-cluster/#installing-the-eventing-component

So we could add docs to enable this if you want to use the default Broker in KFServing?

Reply:
cool! Let's add a note so that existing user knows that you can still get back previous default broker by installing this.